modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,17 +1,65 @@
 // Module included in the following assemblies:
 //
-// virt/install/virt-installing-virtctl.adoc
+// virt/install/virt-enabling-virtctl.adoc
 
 [id="virt-installing-virtctl-client_{context}"]
 = Installing the virtctl client
 
-Install the `virtctl` client from the `kubevirt-virtctl` package.
+Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+
+.Prerequisites
+
+* You must have downloaded the `virtctl` client.
 
 .Procedure
 
-* Install the `kubevirt-virtctl` package:
+* For Linux:
+
+. Extract the tarball. The following CLI command extracts it into the same directory as the tarball and is applicable for all distributions:
++
+[source,terminal]
+----
+$ tar -xvf <virtctl-version-distribution.arch>.tar.gz
+----
+
+. Navigate the extracted folder hierachy and make the `virtctl` binary executable:
++
+[source,terminal]
+----
+$ chmod +x <virtctl-file-name>
+----
+
+. Move the `virtctl` binary to a directory on your PATH.
```

Comment:
s/directory on your PATH/directory in your `$PATH` environment variable

Same comment applies to the Windows and macOS steps as well